app/assets/javascripts/repositories/repository_datatable.js

```diff
@@ -214,6 +214,55 @@ setTimeout(function() {
 // Enables noSearchHidden plugin
 $.fn.dataTable.defaults.noSearchHidden = true;
 
+$('form#form-export').submit(function(e) {
+  var form = this;
+
+  if (currentMode === 'viewMode') {
+    // Remove all hidden fields
+    $('#form-export').find('input[name=row_ids\\[\\]]').remove();
+    $('#form-export').find('input[name=header_ids\\[\\]]').remove();
+
+    // Append samples
+    appendSamplesIdToForm(form);
+
+    // Append visible column information
```

Comment:
If you did seperate function forr appending record IDs, you can/should do the same here for visible columns.